pkg/cmd/rollbackcopy/rollbackcopy.go

```diff
@@ -0,0 +1,102 @@
+package rollbackcopy
+
+import (
+	"context"
+	"fmt"
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+	"io"
+	"k8s.io/klog"
+	"math/rand"
+	"os"
+	"time"
+)
+
+type rollbackCopyOpts struct {
+	checkLeadershipInterval time.Duration
+	rollbackCopyInterval    time.Duration
+	configDir               string
+	errOut                  io.Writer
+}
+
+const (
+	leaderShipCheckInterval time.Duration = 5 * time.Minute
+	rollbackSaveInterval    time.Duration = 30 * time.Minute
```

Comment:
these should reflect what we are going to ship, which is 1 hour

Reply:
4.4 we can use other values for testing this PR is what will merge.

Reply:
Pushed new update.